src/octopeer-github/main/Settings/Settings.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Created by Mitchell on 12-5-2016.
+ */
+
+/**
+ * Class from which user settings can be retrieved.
+ * These settings have been maintained by the front-end.
+ */
+class Settings {
```

Comment:
I don't see this in that commit? I see you've resolved other feedback though, so that's good :)